lit_tests/kernel/wave/codegen.py

```diff
@@ -348,7 +348,7 @@ def mma(
         # CHECK:            %[[D5:.+]] = vector.load %[[D0]][%[[D4]], %[[C0]]] : memref<64x16xf16, strided<[16, 1], offset: ?>>,
         # CHECK-SAME:         vector<4xf16>
         # CHECK:            %[[ALLOC:.+]] = memref.alloc() : memref<32x16xf16, #[[GPU:.+]].address_space<workgroup>>
-        # CHECK:            vector.store %[[D5]], %[[ALLOC]][%[[D4]], %[[C0]]] : memref<32x16xf16,
+        # CHECK:            vector.store %[[D5]], %[[ALLOC]][%[[D2]], %[[C0]]] : memref<32x16xf16,
```

Comment:
Just curious, why would there be a change in LIT test? I feel like the refactoring just cleans it up/change the location of this transformation?

Reply:
So this was a bug. The shared memory store indices should not have any workgroup dependence but they did.

Reply:
Makes sense :)